### PR TITLE
Specify the quanitity of each role

### PR DESCRIPTION
**Pull request by btm**
https://github.com/chef/chef/pull/2950#issuecomment-76869345

### CODE REVIEW

**Comment by thommay** [comment]
:+1: 

There can be only one

**Comment by nathenharvey** [comment]
:+1:

**Comment by jonlives** [comment]
:+1:

**Comment by coderanger** [comment]
:+1:

File: rfc030-maintenance-policy.md [hunk 1]
```diff
@@ -39,6 +41,7 @@ This file is the canonical source for how the Chef project is maintained.
 
 ## Maintainer
 
+* Each component may have mulitple Maintainers
```
Comment (thommay) [inline rfc030-maintenance-policy.md]: "multiple"

**Comment by adamhjk** [comment]
I am the one true king of Gondor, and I approve this message. /cc @chef/rfc-editors 

The king of Gondor requires fixing the typo that @thommay just saw.

**Comment by btm** [comment]
fixed typo, rebased, and merged as f302dd3921376a0da137f10936144d9c8a253b32.

**Pull request closed without merging**